docs/usage/bot-comparison.md

```diff
@@ -0,0 +1,104 @@
+
+This is not meant to be a “versus” or anti-Dependabot page, and it's not marketing.
+It should be as objective as possible, to help users decide which tool is best for them.
+
+If you see anything wrong on this page, please let us know by creating a [Discussion](https://github.com/renovatebot/renovate/discussions), or edit this page with a PR.
+
+## Table
+
+| Name of feature                           | Dependabot                                                                                                                                                                   | Renovate                                                                                                 |
+| ----------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| Dependency Dashboard                      | No                                                                                                                                                                           | Yes                                                                                                      |
+| Grouped updates                           | Yes, create [`groups`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups) manually     | Yes, use community-provided groups, or create your own                                                   |
+| Upgrades common monorepo packages at once | No                                                                                                                                                                           | Yes                                                                                                      |
+| Supported platforms                       | GitHub only                                                                                                                                                                  | GitHub, GitLab, Bitbucket, Azure, Gitea, see [full list](https://docs.renovatebot.com/modules/platform/) |
+| Supported languages                       | [List for Dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#supported-repositories-and-ecosystems) | [List for Renovate](https://docs.renovatebot.com/modules/manager/)                                       |
+| Customizability                           | Low                                                                                                                                                                          | High                                                                                                     |
```

Comment:
Maybe it'll be too much to ask, but it'd help to break this down further. 

What can we customize in Renovate that we can't in Dependabot?